validator_client/src/http_api/tests.rs

```diff
@@ -455,6 +493,76 @@ fn invalid_pubkey() {
     });
 }
 
+#[test]
+fn routes_with_invalid_auth() {
+    let runtime = build_runtime();
+    let weak_runtime = Arc::downgrade(&runtime);
+    runtime.block_on(async {
+        ApiTester::new(weak_runtime)
+            .await
+            .test_with_invalid_auth(|client| async move { client.get_lighthouse_version().await })
```

Comment:
Just a style thing, but I think we can avoid the `async`/`await` wrapping

```suggestion
            .test_with_invalid_auth(|client| client.get_lighthouse_version())
```

Reply:
As discussed, I couldn't make this guy work :(